src/server_manager/test.webpack.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Outline Authors
```

Comment:
Is this file ever used by webpack?

Reply:
We reference it in our Karma config as a file preprocessor (necessary so that we apply our Sentry electron plugin). But I like your suggestion below of removing this file!